wallets/dashcore/send-receive.rst

```diff
@@ -147,3 +150,8 @@ addresses**. In the same way, sending addresses can be seen by clicking **Window
 
 You can edit an existing address by right clicking and selecting **Edit**
 address from the context menu. 
+
+.. figure:: img/dash resend transaction.png
```

Comment:
Let's rename the file to "resend-transaction.png". We can't use spaces in the filename and the others use hyphens.
```suggestion
.. figure:: img/resend-transaction.png
```

Reply:
@rmdash The image file needs to be renamed also to match this

Reply:
Interesting.. i did this already but it reverted. Anyway...renamed it again.